vagrant/README.md

```diff
@@ -6,3 +8,16 @@ In order to disable the rsync option in your Vagrantfile, you need to use this s
 config.vm.synced_folder ".", "/home/vagrant/sync", disabled: true
 ~~~
 
+##Setup VirtualBox shared folder at /vagrant
+
+Install vagrant-vbguest
+
+~~~bash
+$ vagrant plugin install vagrant-vbguest
+~~~
+
+Add this to Vagrantfile
+
+~~~ruby
+config.vm.synced_folder ".", "/vagrant"
```

Comment:
This can fixed in the build script of the Vagrant box i.e. https://github.com/CentOS/sig-cloud-instance-build/blob/master/vagrant/do_vagrant_cbs.sh#L16